### PR TITLE
Add com.corgibytes;versions-maven-plugin;2.11.1

### PR DESCRIPTION
**Pull request by Vel1khan**

### CODE REVIEW

**Comment by Vel1khan** [comment]
I just realised that this is a fork of an already documented reproducible package. Should we keep it?

**Comment by hboutemy** [comment]
good catch
let's keep it, it's not the first time, and the fact that it's a fork does not change anything against the initial objective = prove that it's reproducible

**Review by hboutemy — CHANGES_REQUESTED** [review]
please also add maven-metadata.xml

File: content/com/corgibytes/versions-maven-plugin/versions-maven-plugin-2.11.1.buildspec [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+gitRepo=https://github.com/corgibytes/versions-maven-plugin
+gitTag=versions-maven-plugin-2.11.1
+
+# Rebuild environment prerequisites
+tool=mvn
+jdk=17
+newline=lf
+
+# Rebuild command
+command="mvn clean package -DskipTests -Dmaven.javadoc.skip -Dgpg.skip"
```
[the remark this answers is not in the document]
Comment (Vel1khan) [reply content/com/corgibytes/versions-maven-plugin/versions-maven-plugin-2.11.1.buildspec]: Huh, did not know that. Seems in this case the build profile was in the parent POM. Would you have any idea on whether it's possible to extract the release profiles directly? Are they always listed like this?

```
        <plugin>
          <groupId>org.apache.maven.plugins</groupId>
          <artifactId>maven-release-plugin</artifactId>
          <version>${maven-release-plugin.version}</version>
          <configuration>
            <!-- do not deploy site but use instructions in README.md -->
            <goals>deploy</goals>
            <arguments>-Pmojo-release</arguments>
            <autoVersionSubmodules>true</autoVersionSubmodules>
            <tagNameFormat>@{project.version}</tagNameFormat>
          </configuration>
        </plugin>
        <plugin>
```

[the remark this answers is not in the document]
Comment (hboutemy) [reply content/com/corgibytes/versions-maven-plugin/versions-maven-plugin-2.11.1.buildspec]: usually, release profile is configured in `maven-release-plugin` (it's a best practice, even if not absolutely necessary)

but usually, i's not done as `argument` but as `releaseProfiles` (= the best practice)

**Review by hboutemy — APPROVED** [review]

**Comment by hboutemy** [comment]
I suppose https://repo.maven.apache.org/maven2/com/corgibytes/mrm/ is also reproducible as a fork of https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/content/org/codehaus/mojo/mrm/README.md

**Comment by Vel1khan** [comment]
> I suppose https://repo.maven.apache.org/maven2/com/corgibytes/mrm/ is also reproducible as a fork of https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/content/org/codehaus/mojo/mrm/README.md



> I suppose https://repo.maven.apache.org/maven2/com/corgibytes/mrm/ is also reproducible as a fork of https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/content/org/codehaus/mojo/mrm/README.md

Yes it is indeed. I will create the buildspec for it.

**Pull request merged**